[Docker] Add command to install frontend-vendor assets & replaced bash with sh

silasmahler opened the pull request:
1) Add command to install frontend-vendor assets
2) Remove Docker "bash -c" Commands, because the environment container isn't supporting bash anymore, therefore you need to pass the container-commands as is.

silasmahler committed:
Update docs.md

1) Add command to install frontend-vendor assets
2) Remove Docker "bash -c" Commands, because the environment container isn't supporting bash anymore, therefore you need to pass the container-commands as is.

lcharette commented:
Any Docker guru want to validate this ?

splitt3r commented:
What about `sh` like in https://github.com/userfrosting/UserFrosting/tree/master/docker? The file and the learn site should be equal.

@Silic0nS0ldier seems to work on some Docker related things in https://github.com/userfrosting/UserFrosting/pull/959. Coul be related.

silasmahler committed:
Update docs.md

Update to sh in regart to: https://github.com/userfrosting/learn/pull/76#issuecomment-494454862

silasmahler commented:
> 1. Add command to install frontend-vendor assets
> 2. Remove Docker "bash -c" Commands, because the environment container isn't supporting bash anymore, therefore you need to pass the container-commands as is.

Tested and adapted 2) to @splitt3r suggestion

Silic0nS0ldier commented:
I'll give this a look over once I've the chance. Most likely after https://github.com/userfrosting/UserFrosting/pull/959 has been handled though (duplicate documentation is bad enough, having them go out of sync only compounds the problem).

Silic0nS0ldier committed:
Added note about state of Docker support

Silic0nS0ldier reviewed (approved):
There is a lot left to improve in terms of Docker support and its documentation (especially keeping it up to date). Within the scope of this PR however, LGTM.
